Automattic/jetpack
# PHPCS: WP Post Widget

#### Changes proposed in this Pull Request:

PHPCS cleanup for WP Post Widget.

#### Jetpack product discussion

Fixes #21990

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Pull down changes to JP local dev.
* In the widget editor, add a `Display WordPress Posts (Jetpack)` widget and configure it. You might use `https://jetpack.com/blog/` as the blog URL.
* Make sure no php/js errors are reported when interacting with the widget from backend & frontend.
